spark-cobol/src/main/scala/za/co/absa/cobrix/spark/cobol/source/DefaultSource.scala

```diff
@@ -53,7 +53,7 @@ class DefaultSource
 
     logger.info(s"Cobrix 'spark-cobol' build ${BuildProperties.buildVersion} (${BuildProperties.buildTimestamp}) ")
 
-    val cobolParameters = CobolParametersParser.parse(parameters)
+    val cobolParameters = CobolParametersParser.parse(new ParametersMap(parameters))
```

Comment:
Maybe naming the container class as `Parameters` would be better, as otherwise we'd be replacing a real `map` by a class but still be calling it `map`.

Reply:
Yeah, `Parameters` looks better. Thanks!